### PR TITLE
feat: deprecate mixins

### PR DESCRIPTION
**Pull request by ferferga**
Mixins no longer exist in Vue 3, so their functionality have been moved to normal utility files.

### CODE REVIEW

File: frontend/pages/fullscreen/playback/index.vue [hunk 1]
```diff
@@ -60,13 +59,10 @@ export default Vue.extend({
...
     ...mapStores(pageStore, playbackManagerStore),
     backdropHash: {
-      get(): string {
-        return (
-          this.getBlurhash(
-            this.playbackManager.getCurrentItem,
-            ImageType.Primary
-          ) || ''
-        );
+      get(): string | undefined {
```
Comment (ThibaultNocchi) [inline frontend/pages/fullscreen/playback/index.vue:62]: I know you liked that style of computing methods, but what about now? Don't you think we could drop the "get" when there's no setter?

**Review by ThibaultNocchi — CHANGES_REQUESTED** [review]

File: frontend/components/Buttons/FilterButton.vue [hunk 2]
```diff
@@ -304,7 +305,7 @@ export default Vue.extend({
         const response = (
           await this.$api.filter.getQueryFiltersLegacy({
             userId: this.auth.currentUserId,
-            parentId: this.$route.params.viewId,
+            parentId: sanitizeHtml(this.$route.params.viewId),
```
[the remark this answers is not in the document]
Comment (ferferga) [reply frontend/components/Buttons/FilterButton.vue:308]: @ThibaultNocchi This fixes https://github.com/jellyfin/jellyfin-vue/security/code-scanning/223

File: frontend/pages/fullscreen/playback/index.vue (hunk 1 above)
Comment (ferferga) [reply frontend/pages/fullscreen/playback/index.vue:62]: @ThibaultNocchi I agree, however not sure how necessary the effort to change them all is if we're near the Vue 3 switch.
Most (if not all) the computed properties would need a rewrite anyway with Composition API.

**Comment by ferferga** [comment]
@jellyfin-bot rebase

File: frontend/pages/fullscreen/playback/index.vue (hunk 1 above)
Comment (ThibaultNocchi) [reply frontend/pages/fullscreen/playback/index.vue:62]: Oh no ofc not all of them at one, but in a general manner when seeing one of them. Like this one ^^

File: frontend/components/Buttons/FilterButton.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (ThibaultNocchi) [reply frontend/components/Buttons/FilterButton.vue:308]: ok, then I'd put a comment to explain exactly that :)

[the remark this answers is not in the document]
Comment (ferferga) [reply frontend/components/Buttons/FilterButton.vue:313]: Good idea 👌

**Comment by ferferga** [comment]
Last commit is a really minor change, I think it's better to let it through an already existing PR instead of opening a new one.

**Pull request merged**